### ISSUE TITLE
BUG: Get Friends of Friends feed request 

### ISSUE THREAD

**Issue by ohld** [notmarek/BeFake#68]
All other endpoints work perfectly but this one shows 401 error. Looks like the endpoint was changed or deprecated.

<img width="875" alt="image" src="https://user-images.githubusercontent.com/5613295/231157431-66e09045-a23d-421b-9f4c-98b2da634cfb.png">

I used Firebase ReCaptcha to login to BeReal. Maybe I need to login to BeReal backend as well (https://mobile.bereal.com/api/).

**Comment by ohld** [comment]
<img width="761" alt="image" src="https://user-images.githubusercontent.com/5613295/231207446-d2a49177-be7e-4a73-8614-10304c97cb2d.png">

Can you please guide me which endpoint should I try to patch? Because I tried to patch /settings and it returned 404.
``` python
res = bf.api_request("patch", f"settings", data={"featureFlags": {'friendsOfFriendsFeed': True}})
```

FYI: When I did GET /settings request with data field (accidentally) all by current feature flags disappeared: 
The new settings looks like this now:
```
bf.api_request("get", f"settings")
{'mandatoryVersions': {'android': '0.28.5', 'ios': '0.27.0'},
 'recommendedVersions': {'android': '0.28.5', 'ios': '0.27.0'},
 'polling': {'feedsFriends': 5000, 'moment': 5000},
 'storage': {'bucket': 'gs://storage.bere.al'}}
```

**Comment by VxlerieUwU** [comment]
```
PUT https://mobile.bereal.com/api/terms/show-friends-to-friends

body = {
  "status": "ACCEPTED"
}

```

**Comment by VxlerieUwU** [comment]
Implemented with d78c3a049a805bf4e8dc7d3af77b66e35f673f46, I'll add the respective command line argument and options later

**Comment by bdteo** [comment]
Hi, it seems they've changed their API, so the solution does not work. I get 401 error even after sending the PUT request.
